### ISSUE TITLE
[blueprints] Fix various issues with blueprints

### ISSUE THREAD

**Issue by sumerjabri** [craftercms/craftercms#5098]
## v4
* Fix Alexa BP (add `ui.xml` latest)
* Fix restaurants where dropping images on the "Chef's" area doesn't reload correctly (you have to refresh)
* Fix ICE in "Simple Brochureware Site"
* Fix Pizza BP (text overlaps and looks broken)

Fix the error:
```
Caused by: org.craftercms.commons.upgrade.exception.UpgradeNotSupportedException: Current version '3.1.4.4' for 'companyshowcase' cannot be upgraded automatically, requires '>=3.1.9'
```

## v3.1
* Fix Wordify BP on v3.1.18

**Comment by jvega190** [comment]
Pizza-bp issues fixed in https://github.com/rivetlogic/craftercms-bp-pizza/pull/25

**Comment by jvega190** [comment]
Simple Brochureware Site ICE issues fixed in https://github.com/craftercms/studio-ui/pull/2665

**Comment by jvega190** [comment]
Restaurant bp fixed in https://github.com/rivetlogic/craftercms-bp-restaurant/pull/39

**Comment by jvega190** [comment]
UpgradeNotSupportedException fixed in all rl-blueprints, PR's in here https://github.com/craftercms/craftercms/issues/5014

**Comment by sumerjabri** [comment]
@jvega190 the link above to 5014 might be incorrect, please double-check.

**Comment by jvega190** [comment]
@sumerjabri I was linking that ticket that has all of the bps PRs, but I'll update all of them to reference this ticket too

**Comment by jvega190** [comment]
@sumerjabri can this be closed for 3.1?

**Comment by sumerjabri** [comment]
Verified for 3.1.x
